src/pages/webhooks/create-webhooks.md

```diff
@@ -1,57 +1,89 @@
+
+Admin field | XML attribute | Description
+--- | --- | ---
+**Webhook Method** | `method.name` | If using the Admin, select one of the supported Commerce webhook names from the dropdown. Otherwise, The value of the webhook method name must be in the form `<event_type>.<webhook_name>`, where `event_type` is either `observer` or `plugin`, and `webhook_name` matches a valid Commerce event name. In PaaS environments, use the `bin/magento webhooks:list:all` command to display a list of possible webhooks.
+**Webhook Type** | `method.type` | Specify whether to run the webhook `before` or `after` the original action.
+**Batch Name** | `batch.name` | A unique name for the batch. Use a descriptive name that encompasses all the hooks in the batch. The name must contain English alphanumeric characters and underscores (_) only.
+**Batch Order** | `batch.order` | An integer that sets the order in which multiple webhooks are executed. All hooks within a batch are sent in parallel. Therefore, as you add hooks to a batch, keep in mind what task each hook will perform. For example, since the hooks are executed in parallel, you should not place a hook that relies on a response from another hook in the same batch. A default value of 0 is saved if no value is set.
+**Hook Name** | `hook.name` | A name that must be unique within a batch. The name must contain English alphanumeric characters and underscores (_) only.
+**Hook Priority** | `hook.priority` | The priority of the merging hook results in the batch. The priority is treated as 0 if a value is not set.
+**URL** | `hook.url` | The HTTP endpoint to send the request for processing. The hook URL is formed by concatenating an environment variable and a partial path. This practice is useful for developing in different environments, such as those for staging and production, where the hook URLs are different.
+**Timeout** | `hook.timeout` | A hard timeout limit (milliseconds) for the request. Requests exceeding this timeout are aborted and logged. The default value of 0 indicates there is no timeout limit.
+**Soft timeout** | `hook.softTimeout` | A soft timeout limit (milliseconds) for the request. Requests exceeding this timeout are logged for debugging purposes.
+**Cache TTL** | `hook.ttl` | The cache time-to-live (in seconds) for requests with the same URL, body, and headers. If this attribute is not specified, or if the value set to `0`, the response is not cached.
+**Fallback error message** | `hook.fallbackErrorMessage` | The error message to display if the hook fails.
+**Required** | `hook.required` | Specifies whether hook execution is required or optional. When set to **Optional** (`false`), if the hook fails to execute, the failure is logged and subsequent hooks continue to be processed. When set to **Required** (`true`), a failure terminates the process.
+**Active** | `hook.remove` | Indicates whether to skip a removed hook during the batch execution.
```

Comment:
The difference between PaaS/SaaS is a little complicated here.

In the Admin, if you set active to No, the hook is skipped
In webhooks.xml, if you set remove to true, the hook is skipped